Assets/Scripts/Interactables/NoisePing.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NoisePing : MonoBehaviour
+{
+    public GameObject pingPrefab;
+    public float pingFloorOffset = 0.5f;
+    public float pingDuration = 0.75f;
+    public float pingGrowthScale = 0.5f;
+    
+    private GameObject pingInstance;
+    
+    public void SpawnNoisePing(Vector3 hitPoint, Vector3 hitNormal)
+    {
+        pingInstance = Instantiate(pingPrefab, hitPoint + (pingFloorOffset * hitNormal), 
+            Quaternion.LookRotation(hitPoint));
+        Destroy(pingInstance, pingDuration);
+    }
+
+    private void FixedUpdate()
+    {
+        if (!pingInstance) return;
+
+        pingInstance.transform.localScale += new Vector3(pingGrowthScale, 0f, pingGrowthScale);
```

Comment:
I think it would be cool to do this in terms of radius size or something like that possibly. I can't as easily conceptualize how the growthScale applying every frame would look like, if that makes sense?

It would just be something like, the distance we go per second would be `pingRadius / pingDuration` and then we instead add to scale, `that-number * Time.fixedDeltaTime` or something like that?